src/figuresClasses.ts

```diff
@@ -1,11 +1,75 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
```

Comment:
Consider using an enum for the shape
```suggestion
  shape: Shape = 'triangle';
```